add-custom-dynamic-entity-data-source/src/main/java/com/esri/arcgismaps/sample/addcustomdynamicentitydatasource/screens/MainScreen.kt

```diff
@@ -74,7 +77,28 @@ fun MainScreen(sampleName: String) {
                         .weight(1f),
                     mapViewProxy = mapViewModel.mapViewProxy,
                     arcGISMap = mapViewModel.arcGISMap,
-                    onSingleTapConfirmed = mapViewModel::identify
+                    onSingleTapConfirmed = mapViewModel::identify,
+                    content = {
+                        mapViewModel.selectedGeoElement?.let { selectedGeoElement ->
+                            Callout(
+                                modifier = Modifier.wrapContentSize(),
```

Comment:
If content is too long it may take up the entire screen width, I would suggest to apply a maxWidth: 

```suggestion
                                modifier = Modifier.sizeIn(maxWidth = 250.dp),
```